esp/esp/users/models/__init__.py

```diff
@@ -2279,6 +2279,7 @@ class Permission(ExpirableModel):
             ("Teacher/Classes/All", "Classes/All"),
             ("Teacher/Classes/View", "Classes/View"),
             ("Teacher/Classes/Edit", "Classes/Edit"),
+            ("Teacher/Classes/CancelReq", "Classes/CancelReq"),
```

Comment:
The second one should be human-readable, e.g. "Request class cancellation".  Not sure why the few before it aren't.